e2e/app/page/cohort-search-page.ts

```diff
@@ -132,7 +132,7 @@ export default class CohortSearchPage extends AuthenticatedPage {
     await finishAndReviewButton.click();
 
     // Click Save Criteria button in sidebar
-    const helpSidebar = new HelpSidebar(this.page);
+    const helpSidebar = new HelpSidebar(this.page, '//*[@id="help-sidebar"]//*[@id="selection-list"]');
```

Comment:
This is not what I meant; this is still problematic because when we inevitably add methods to the HelpSidebar component which do non-cohorts things (like opening the data dictionary), they will just fail with this Xpath. Likewise, in other contexts the cohorts methods will not work.

Reply:
It would be more involved and more refactoring. `await SelectMenu.findByName(this.page, {ancestorLevel: 0}, this);` or `buildXPath` function take a `Container` object for Parent instead of xpath string.

Reply:
Hmm I see. btw, should xPathOptions support specifying a specific xpath as an option? I was looking for this on my last PR, but it seems it would have been useful here as well.

Please leave a TODO here or on help-sidebar and then perhaps @dolbeew has other ideas about how to clean this up. If left as-is, someone is going to try to reuse this component later and get an unfortunate surprise.